examples/React-Native/src/welcome.tsx

```diff
@@ -0,0 +1,23 @@
+import { StyleSheet, Text, View } from 'react-native';
+import { useBoolVariation } from '@launchdarkly/react-native-client-sdk';
+
+export default function Welcome() {
+  //TODO Set my-boolean-flag to a valid boolean flag key in your project/environment.
+  const flagValue = useBoolVariation('bool-flag', false);
```

Comment:
`'bool-flag'` should be either a constant outside the scope of the `Welcome` function, or an environment variable.